src/main/java/core/basesyntax/Robot.java

```diff
@@ -1,4 +1,58 @@
 package core.basesyntax;
 
 public class Robot {
+    private Direction direction;
+    private int coordinateX;
+    private int coordinateY;
+
+    public Robot(Direction direction, int coordinateX, int coordinateY) {
+        this.direction = direction;
+        this.coordinateX = coordinateX;
+        this.coordinateY = coordinateY;
+    }
+
+    public Direction getDirection() {
+        return direction;
+        // текущее направление взгляда
+    }
+
+    public int getCoordinateX() {
+        return coordinateX;
+        // текущая координата X
+    }
+
+    public int getCoordinateY() {
+        return coordinateY;
+    }
+
+    public void turnLeft() {
+        switch (direction) {
+            case UP: direction = Direction.LEFT;
```

Comment:
@EliosPride missed 'break' for each case

Reply:
@al-pavlenko added 'break', done